### PR TITLE
Completed javascript-conditionals exercise -Julie Luangpraxay

### PR DESCRIPTION
**Pull request by julieluangpraxay**
Screenshot 2023-07-12 at 8 50 59 PM

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+console.log('isEven value:', isEven(4));
+
+function startsWithJ(string) {
+  if (
+    string.startsWith('J') &&
+    string.charAt(0) === string.charAt(0).toUpperCase()
+  ) {
+    return true;
+  }
+}
```
Comment (shawnkost) [inline javascript-conditionals/main.js:37]: We should also be `return`ing `false` if it doesn't start with a 'J'

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+console.log('isOldEnoughToDrinkandDrive value:', isOldEnoughToDrinkAndDrive());
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 0 && pH <= 7) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:77]: This should be greater than or equal to 0

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple tweaks here Julie!

File: javascript-conditionals/main.js (hunk 1 above)
Comment (julieluangpraxay) [reply javascript-conditionals/main.js]: Screenshot 2023-07-13 at 11 38 31 AM

File: javascript-conditionals/main.js (hunk 2 above)
Comment (julieluangpraxay) [reply javascript-conditionals/main.js]: Screenshot 2023-07-13 at 11 40 17 AM

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -76,7 +76,7 @@ console.log('isOldEnoughToDrinkandDrive value:', isOldEnoughToDrinkAndDrive());
 function categorizeAcidity(pH) {
   if (pH === 7) {
     return 'neutral';
-  } else if (pH >= 0 && pH <= 7) {
+  } else if (pH <= 0 && pH <= 7) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:79]: This should be greater than or equal to, not less than or equal to.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Julie!

File: javascript-conditionals/main.js (hunk 3 above)
Comment (julieluangpraxay) [reply javascript-conditionals/main.js]: Screenshot 2023-07-13 at 10 50 16 PM

sorry! I feel dyslexic with greater than or less than sometimes...

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**